packages/teleport/src/lib/tdp/sharedDirectoryManager.ts

```diff
@@ -81,6 +81,26 @@ export class SharedDirectoryManager {
     return infos;
   }
 
+  // Reads length bytes starting at offset from a file at path.
+  async readFile(
+    path: string,
+    offset: bigint,
+    length: number
+  ): Promise<Uint8Array> {
+    this.checkReady();
+
+    const fileHandle = await this.walkPath(path);
```

Comment:
These awaits should probably be wrapped with a try/catch

Reply:
Makes sense as a first impression, but actually I want them to just throw that up to the `client.ts::Client`